app/Models/Post.php

```diff
@@ -74,6 +76,19 @@ public function user(): \Illuminate\Database\Eloquent\Relations\BelongsTo
         ]);
     }
 
+    /**
+     * Belongs To An Updated User.
+     *
+     * @return \Illuminate\Database\Eloquent\Relations\BelongsTo<User, $this>
+     */
+    public function updatedBy(): \Illuminate\Database\Eloquent\Relations\BelongsTo
+    {
+        return $this->belongsTo(User::class, 'updated_by', 'id')->withDefault([
+            'username' => 'System',
+            'id'       => User::SYSTEM_USER_ID,
+        ]);
```

Comment:
Let's remove the `withDefault` here and add a `withoutTrashed`

Reply:
Done